charts/dispatch/charts/identity-manager/templates/secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
See #146. We may want to make secret definition an explicit prerequisite to chart installation.  That way no sensitive information is required to go into a values.yaml file, and we don't need to worry about credentials leaking to screen.  We can also handle this in our install command instead of relying on charts... thoughts?